experimental/packages/opentelemetry-instrumentation-grpc/test/helper.ts

```diff
@@ -836,5 +927,66 @@ export const runTests = (
         runClientMethodTest(method);
       });
     });
+
+    describe('Test capturing metadata', () => {
```

Comment:
I would prefer to have one test with grpc and grpc-js so we are sure this work with each version